helm-chart/templates/analytics/ingress.yaml

```diff
@@ -1,48 +1,38 @@
 {{- $version := .Capabilities.KubeVersion.GitVersion -}}
 {{- if and .Values.ingress.enabled .Values.analytics.enabled -}}
 {{- $fullName := include "cvat.fullname" . -}}
-{{- if semverCompare ">=1.19-0" $version -}}
 apiVersion: networking.k8s.io/v1
-{{- else if semverCompare ">=1.14-0" $version -}}
-apiVersion: networking.k8s.io/v1beta1
-{{- else -}}
-apiVersion: extensions/v1beta1
-{{- end }}
 kind: Ingress
 metadata:
   name: {{ $fullName }}-analytics
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "cvat.labels" . | nindent 4 }}
   annotations:
+    {{- if .Values.traefik.enabled }}
     traefik.ingress.kubernetes.io/router.middlewares: {{ $.Release.Namespace }}-stripprefix@kubernetescrd,
       {{ $.Release.Namespace }}-forwardauth@kubernetescrd
-    {{- with .Values.analytics.ingress.annotations }}
+    {{- end }}
+    {{- with .Values.ingress.annotations }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
-{{- if semverCompare ">=1.19-0" $version }}
-  ingressClassName: {{ .Values.ingress.className }}
+{{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
 {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
-    - host: {{ .host | quote }}
+    {{- if .Values.ingress.hostname }}
```

Comment:
This seems odd. If `ingress.hostname` is not defined, then the created ingress will have no rules - but then what's the point of creating the ingress in the first place? Seems like `ingress.hostname` should be required if `ingress.enabled` is true.